wzdx/tools/polygon_tools.py

```diff
@@ -126,6 +126,35 @@ def get_road_direction_from_coordinates(coordinates):
     return direction
 
 
+# function to get road direction by using geometry coordinates
+def get_road_directions_from_coordinates(coordinates):
```

Comment:
Is this direction supposed to line up with the overall labeled road direction? If so, eventually we'll have to revisit this logic. There's some odd places where this wouldn't hold up - for instance if the coordinates happen to be in a bend in the road for shoulder work, or a large section of highway tending northward that is actually on a road labeled for westbound traffic.

Reply:
yeah, it does its best but can easily fail. It is just for worst case scenarios, in iCone messages where the direction is not specified, and I need to choose one